src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+/* eslint-disable */
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  let isNoComments = false;
+
+  if (post.comments.length === 0) {
+    isNoComments = true;
+  }
```

Comment:
The `isNoComments` variable can be simplified by directly using the condition `post.comments.length === 0` in the JSX. This will make the code cleaner and more concise.